cmd/verify/verify.go

```diff
@@ -95,97 +95,97 @@ func Command() *cobra.Command {
 		&verifyConcurrency,
 		"concurrency",
 		0,
-		"number of tables to process at a time (defaults to number of CPUs)",
+		"Number of tables to process at a time (defaults to number of CPUs).",
 	)
 	cmd.PersistentFlags().IntVar(
 		&verifyTableSplits,
 		"table-splits",
 		1,
-		"number of shards to break down each table into whilst doing row-based verification",
+		"Number of shards to break down each table into while doing row-based verification.",
 	)
 	cmd.PersistentFlags().IntVar(
 		&verifyRowBatchSize,
 		"row-batch-size",
 		20000,
-		"number of rows to get from a table at a time",
+		"Number of rows to select from a table at a time.",
 	)
 	cmd.PersistentFlags().IntVar(
 		&verifyLimitRowsPerSecond,
 		"rows-per-second",
 		0,
-		"if set, maximum number of rows to read per second during scanning per shard",
+		"If set, maximum number of rows to read per second during scanning per shard.",
 	)
 	cmd.PersistentFlags().BoolVar(
 		&verifyFixup,
 		"fixup",
 		false,
-		"whether to fix up any rows",
+		"Whether to fix up any rows.",
 	)
 	cmd.PersistentFlags().DurationVar(
 		&verifyContinuousPause,
 		"continuous-pause-between-runs",
 		0,
-		"time to pause between continuous runs",
+		"Time to pause between continuous runs.",
```

Comment:
What are the units of time?

```suggestion
		"Amount of time to pause between continuous runs.",
```

Reply:
This can actually be specified as a string, so like `1h`, `2m`, etc.

Reply:
```Amount of time to pause between continuous runs (e.g., 1h, 2m).```

Reply:
Good let's do it this way!